packages/trpc/src/procedures/labels/createLabel.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { trpcSetup, tearDown } from "../../testutils";
+import { prisma } from "@recipesage/prisma";
+import { User } from "@prisma/client";
+import type { CreateTRPCProxyClient } from "@trpc/client";
+import type { AppRouter } from "../../index";
+
+describe("create Label", () => {
```

Comment:
Can we name these describe blocks the same as the method being tested (in this case `createLabel`)